### PR TITLE
This is to update Windows profile with a new root certificate

### PR DESCRIPTION
**Pull request by seb-szy**

### CODE REVIEW

**Comment by robinmitra** [comment]
@seb-szy Could you please add more details to the PR as I don't context around this change?

**Comment by seb-szy** [comment]
This PR was prepared with Rafal ahead of certificate rotation, I did not notice request for further clarification. Within change we added an additional thumbprint for a new certificate.

File: source/assets/govwifi-profile.xml [hunk 1]
```diff
@@ -44,6 +44,7 @@
 										<TrustedRootCA>de 28 f4 a4 ff e5 b9 2f a3 c5 03 d1 a3 49 a7 f9 96 2a 82 12 </TrustedRootCA>
 										<TrustedRootCA>32 3c 11 8e 1b f7 b8 b6 52 54 e2 e2 10 0d d6 02 90 37 f0 96 </TrustedRootCA>
 										<TrustedRootCA>03 9e ed b8 0b e7 a0 3c 69 53 89 3b 20 d2 d9 32 3a 4c 2a fd </TrustedRootCA>
```
Comment (phantas) [inline source/assets/govwifi-profile.xml]:
```suggestion
```
Remove this, as we no longer need `GeoTrust Primary Certification Authority - G3` in our certificate trust chain

File: source/assets/govwifi-profile.xml [hunk 2]
```diff
@@ -44,6 +44,7 @@
 										<TrustedRootCA>de 28 f4 a4 ff e5 b9 2f a3 c5 03 d1 a3 49 a7 f9 96 2a 82 12 </TrustedRootCA>
 										<TrustedRootCA>32 3c 11 8e 1b f7 b8 b6 52 54 e2 e2 10 0d d6 02 90 37 f0 96 </TrustedRootCA>
```
Comment (phantas) [inline source/assets/govwifi-profile.xml]: Do we know which certificates are these? They don't correspond to any of the intermediate certs. Just curious if they still apply

Comment (seb-szy) [reply source/assets/govwifi-profile.xml]: Two unaccounted root certs were there already so we just added a new one prior to the cert rotation.

**Pull request merged**